README.md

```diff
@@ -43,8 +42,6 @@ Then pin names need to be configured as instructed in the drivers README file.
 
 To connect the ESP8266 module to your development board, look at the [ESP8266 Cookbook page](https://developer.mbed.org/users/4180_1/notebook/using-the-esp8266-with-the-mbed-lpc1768/). In general, this means hooking up the ESP8266 TX pin to `D0` and the ESP8266 RX pin to `D1` on your development board.
 
-**Note:** On NUCLEO development boards, pins `D0` and `D1` are used for serial communication with the computer. Use pins `D8` (to ESP8266 TX) and `D2` (to ESP8266 RX) instead.
```

Comment:
Actually, is this still valid, even though we remove the L476RG?

Reply:
Theoretically 429ZI should work with the default pins, although I must say I wasn't able to get the wifi example running with either D1/D0 or D8/D2 configuration (I tried both with Grove's board plugged into the Arduino pins and with just 3v3,GND,Tx and RX wired to the accurate pins). 
A Jenkins config would be very helpful here, to compare my setup.